src/Core/View.js

```diff
@@ -180,6 +182,8 @@ function _preprocessLayer(view, layer, provider, parentLayer) {
         defineLayerProperty(layer, 'visible', true);
         defineLayerProperty(layer, 'opacity', 1.0);
         defineLayerProperty(layer, 'sequence', 0);
+
+        layer.transform = feature2Texture;
```

Comment:
layer.type color is also used for simple texture coming from WMTS, TMs and WMS, btw. It will still work with that, but it might be a bit confusing.

Reply:
Maybe then we could change the type if we want to use `feature2Texture`, something like `raster` (and then drop the `RasterProvider`)

Reply:
It's a bit confusing because I feel like we're mixing input and output type. Regular color layers are `color` all the way, whereas in our case, the input type is `vector`, but the output type is `color`... 

Maybe just have:
```js

  if (layer.type == 'color') {
    layer.transform = feature2Texture;
  } else if (layer.type == 'geometry') {
    layer.transform = feature2Mesh;
  }
```  
in WFSProvider.preprocess ? 

After all, the input format is determined - or at least constrained - by the protocol, and it is only currently supported for WFS layer. 

Or another idea (that'll take a bit more work maybe), make the `preprocessing` phase marks each layer with the output format of the protocol. So for layers with protocol set to 'raster', 'wfs' (and '3dtiles' in the future?), that would be `layer.protocolOutputFormat = 'feature'` (ok maybe a better name needs to be found), and for WMS that would be `'texture'` for instance. Then in View.js:178, you'd have:

```js
if (layer.protocolOutputFormat  == 'feature') {
   // logic to set the right layer.transform according to the type
}
```

Also the advantage is that it'll be easy to have this property set by other providers like `FileProvider`, which would be one good step (maybe the final one actually) towards the removal of RasterProvider (parsers would return Feature instead of Mesh directly). WDYT?